controllers/suite_test.go

```diff
@@ -178,3 +178,16 @@ func newPod(owner client.Object, scheme *runtime.Scheme) *corev1.Pod {
 	}
 	return p
 }
+
+func newNoCacheClient(cache cache.Cache, config *rest.Config, options client.Options, uncachedObjects ...client.Object) (client.Client, error) {
+	c, err := client.New(config, options)
+	if err != nil {
+		return nil, err
+	}
+
+	return client.NewDelegatingClient(client.NewDelegatingClientInput{
+		CacheReader:     nil,
```

Comment:
```suggestion
```

Isn't this the zero-value?

Reply:
updated to just disable caching Job objects